.gitignore

```diff
@@ -181,3 +181,14 @@ build-iPhoneSimulator/
 /sandbox/plc4go/temp/
 /plc4go/.idea/
 plc4j/examples/hello-storage-elasticsearch/.factorypath
+
+
+*.make
+*.cmake
+*.internal
+link.txt
+Makefile
```

Comment:
```suggestion
```
This would actually remove the plc4go "wrapper" Makefile from plc4gp again so a more specific ignore would be good. (Not sure why the Makefile is ignored here)

Reply:
The makefiles in plc4c are generated using cmake. I have changed it to only ignore plc4c makefiles.